### DISCUSSION TITLE
Embeddable Unique Visitors Counter (issue #22)

### DISCUSSION THREAD

**Discussion by paidlowered** [Litlyx/litlyx#23] [Ideas]
Wishlist:
88x31 button (![proxmox](https://github.com/user-attachments/assets/7ee33fe2-d308-409d-8542-a6d60104310c)![caddy](https://github.com/user-attachments/assets/c439d8ae-bc26-4e7d-9a39-ee47fa63b036) Could display unique visits, total visits, or bounce rate
Large widget that tracks visits by country
Graph of visits over time

I know this is already an issue, it’s just you told me to bring this to the discussions tab.

**Comment by AntonioVerdiglione1996** [comment]
Yes here is the perfect place to talk about it. We will update you soon!
